src/Validation/DNSRecords.php

```diff
@@ -8,12 +8,12 @@ class DNSRecords
     /**
      * @var array $records
      */
-    private $records = [];
+    private $records;
```

Comment:
Please keep it as an empty array.

Reply:
The initializer is redundant as the constructor immediately overwrites it.
We can either drop the redundant initializer or to make things more explicit, we could make this `private array $records` and drop the comment above which will also be redundant then.

Reply:
Good point. I like the explicit typing. Let's go with that solution.